config/stage.json

```diff
@@ -72,6 +72,7 @@
 		"post-editor/html-toolbar": true,
 		"post-editor/insert-menu": true,
 		"post-editor/image-editor": true,
+		"post-editor/mentions": true,
```

Comment:
Feature flags happen to default to false if not available, but this doesn't hold true to other config values like keys. To be safe, I usually like explicitly adding this to each config file.